### ISSUE TITLE
Focus is lost after interacting with "Auto Update" checkbox using keyboard:A11y_VisualStudioCode_Extensions_Keyboard

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/vscode#318700]
Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification.
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## GitHub Tags
#A11yTCS;#Visual Studio Code Client; #E2E_VisualStudioCodeClient_Win32_Dec2025;#DesktopApp;#FTP;#Win32;#A11ySev3; #A11yMAS; #WCAG2.4.3; #Keyboard; #SCIM:Bug5038576;

## Issue Subcategory 
Focus is incorrect or lost after dynamic interaction 

## User Experience 
Keyboard users lose their place after closing a popup, deleting a list item, or completing an inline action. 

## Repro Steps 
Launch Visual Studio Code > Extension > Search for Microsoft sentinel extension >Auto Update checkbox > Observe issue: Focus is lost after interacting with "Auto Update" checkbox using keyboard.

## Actual Result 
After activating the Auto Update checkbox, focus moves away from the control and becomes not visible. Users lose track of current focus location. Pressing Tab moves focus to the next control.

## Expected Result 
Focus should remain on the Auto Update checkbox after it is checked/unchecked, and the focus indicator should remain clearly visible.

## How to Fix 
Manage focus programmatically: return it to the trigger element after dialog close, or to the next sibling item after deletion. Never let focus reset to <body> or to some illogical element after dynamic content changes.

## Environment details 
Application Name: VS  Code
Version: 1.121.0
Edition Windows 11 Enterprise
Version 25H2
OS build 26200.8390

## Attachment

https://github.com/user-attachments/assets/cc158dfb-6358-4805-a147-a68e51e66df0

**Comment by kapilvaishna** [comment]
Screen reader behavior

 <img width="1280" height="768" alt="Image" src="https://github.com/user-attachments/assets/e14b702a-82d8-41f6-b5b4-ceb5cc15ba9f" />

**Comment by Mana-swi** [comment]
#a11ydb-focus-lost-on-action;
